src/OrbitBase/include/OrbitBase/GetProcessIds.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_GET_PROCESS_IDS_H
+#define ORBIT_BASE_GET_PROCESS_IDS_H
+
+#include <sys/types.h>
+
+#include <vector>
+
+namespace orbit_base {
+
+#if defined(__linux)
+// Get the process ids of all currently running processes.
+std::vector<pid_t> GetAllPids();
+
+// Get the thread ids of all the threads belonging to process 'pid'.
+std::vector<pid_t> GetTidsOfProcess(pid_t pid);
+
+// Get all thread ids of all currently running processes.
```

Comment:
```suggestion
// Get all thread ids of all currently running threads.
```

Reply:
Went with the even more explicit

"Get all thread ids of all the threads in all currently running processes."